frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -89,7 +89,7 @@ object Interpreter {
     logger.info(s"$header$line")
     logger.info(s"$t$bloopName is made with love at the Scala Center <3$line")
     logger.info(s"$versions$line$line") // I have no idea why two are required, one is not enough
-    logger.info(s"${t}It is maintained by $developers.")
+    logger.info(s"${t}It is maintained by $developers")
```

Comment:
Just curious, what's wrong with having a dot at the end of sentences?

Reply:
Either variant is fine as long as we use punctuation consistently. Right now, periods appear in some log messages, but not all.

Based on your remark, I did some more research and there appears to be a [concensus](https://ux.stackexchange.com/questions/18671/periods-at-the-end-of-a-sentence-in-alert-message) to favour punctuation in log messages, but not in titles. I will add them back.

Reply:
Thanks for the link, that was interesting! We should indeed be consistent here, so you did a good job at trying to consolidate one approach. Do you by any chance know how `npm` and `cargo` do this? I think they don't use dots there. It may be interesting dropping the dots in favor of whatever the majority of build tools in other language communities do!

Reply:
npm does not have the best UI and they do not use periods consistently either. Cargo does not use periods in messages ([screenshot](https://user-images.githubusercontent.com/15943089/36560466-a4e31226-17c5-11e8-807c-b9dd62cbcc39.png)).